modal/modal.js

```diff
@@ -117,7 +117,7 @@ document.addEventListener('DOMContentLoaded', () => {
   var blockedUrlsPrefix = [
     'about:',
     'chrome:',
-    'moz-extension:',
```

Comment:
I normally add a trailing comma, so when adding more items, `git` doesn't show irrelevant changes.

Reply:
I usually do the same but eslint is configured this way. Maybe we could change the rule if @Mte90 agrees (and we could enforce the usage of const/let instead of var too)